js/plugins-menu.js

```diff
@@ -0,0 +1,4 @@
+//Opens Upgrade link in plugins menu as new tab.
+jQuery(document).ready( function($) {
```

Comment:
This could probably be made an anonymous function; in conjunction to enqueueing in the footer.

Reply:
Done.